kernelci/config/test.py

```diff
@@ -278,7 +278,8 @@ class RootFS(_YAMLObject):
     """Root file system model."""
 
     def __init__(self, fs_type, boot_protocol='tftp', root_type=None,
-                 prompt="/ #", params=None, ramdisk=None, nfs=None):
+                 prompt="/ #", params=None, ramdisk=None,
+                    nfs=None, diskfile=None):
```

Comment:
One last tiny thing :sweat_smile: 
```suggestion
                 prompt="/ #", params=None, ramdisk=None,
                 nfs=None, diskfile=None):
```
```
kernelci/config/test.py:282:21: E127 continuation line over-indented for visual indent
```

Reply:
I very much appreciate the inputs. Thanks.

I have pushed a update.